python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -361,8 +368,17 @@ async def main() -> None:
             pass
         elif isinstance(task, str):
             first_chat_message = TextMessage(content=task, source="user")
-        elif isinstance(task, TextMessage | MultiModalMessage | StopMessage | HandoffMessage):
+        elif isinstance(task, (TextMessage, MultiModalMessage, StopMessage, HandoffMessage)):
             first_chat_message = task
+        elif isinstance(task, list):
+            if not task:
+                raise ValueError("Task list cannot be empty")
+            if not all(isinstance(msg, (TextMessage, MultiModalMessage, StopMessage, HandoffMessage)) for msg in task):
+                raise ValueError("All messages in task list must be valid ChatMessage types")
+            first_chat_message = task[0]
```

Comment:
This part still needs to be updated. Should be similar to `BaseChatAgent`.

Reply:
We're using concrete message types for type checking instead of Annotated type i.e. `ChatMessage`, which I believe is the correct implementation and aligns the message validation with how `BaseChatAgent` handles them. Please let me know if you are thinking of a different approach.

Reply:
Okay, I have updated it to use Annotated type validation.